api/src/http.rs

```diff
@@ -289,7 +289,7 @@ pub struct BlobCacheEntry {
     pub domain_id: String,
     /// Deprecated: data prefetch configuration: BlobPrefetchConfig.
     #[serde(default)]
-    pub fs_prefetch: Option<BlobPrefetchConfig>,
+    pub blob_prefetch: Option<BlobPrefetchConfig>,
```

Comment:
Seems the field has been deprecated?

Reply:
I suppose the deprecated one is BlobPrefetchConfig sturct ?

Reply:
I think it says that we should use `blob_config.prefetch_config` instead of `fs_prefetch`.

Reply:
`BlobCacheEntry` is newly introduced and v2.1 is not released. Why deprecate a never released config?

Reply:
cc @jiangliu PTAL.

Reply:
I would like to remove this field and makes use of the `blob_config.prefetch_config`. When I was refactering the code, `fs_prefetch` is kept for backward compatibility. Now we are going to change its name, so breaking backward compatibility. Then how about just removing it?

Reply:
Agree :-)

Reply:
Um. I suppose it won't break backward compatibility since the struct `BlobCacheEntry` is added in API/v2. We can just safely remove `fs_prefetch` from `BlobCacheEntry`

Reply:
@jiangliu It is removed cc @imeoer